ogusa/TPI.py

```diff
@@ -523,8 +523,11 @@ def run_TPI(p, client=None):
             lazy_values.append(
                 delayed(inner_loop)(guesses, outer_loop_vars,
                                     initial_values, j, ind, p))
-        results = compute(*lazy_values, scheduler=dask.multiprocessing.get,
-                          num_workers=p.num_workers)
+        with Client(direct_to_workers=True) as c:
+            futures = c.compute(lazy_values, scheduler=dask.multiprocessing.get,
+                                num_workers=p.num_workers)
+            result = c.gather(futures)
```

Comment:
```suggestion
            results = c.gather(futures)
```

Whoops, this should have been `results`.